common/tier4_autoware_utils/test/src/geometry/test_geometry.cpp

```diff
@@ -874,45 +874,102 @@ TEST(geometry, calcInterpolatedPose)
   }
 
   // Quaternion Interpolation
-  {
-    geometry_msgs::msg::Pose src_pose;
-    src_pose.position = createPoint(0.0, 0.0, 0.0);
-    src_pose.orientation = createQuaternionFromRPY(deg2rad(0), deg2rad(0), deg2rad(0));
+  {{geometry_msgs::msg::Pose src_pose;
```

Comment:
Unusual shape, but CI has passed...hmm

Reply:
He asked me about that in another PR. Removing one pair of `{}` can solve this format collapse.

Reply:
Fixed